lib/galaxy/config/config_manage.py

```diff
@@ -691,6 +692,14 @@ def _build_sample_yaml(args, app_desc):
     _write_to_file(args, f, destination)
 
 
+def _overwrite_uwsgi_defaults():
+    # Prevents runtime error in logging module under Python 3.7.1+
```

Comment:
Can you add a note to the schema as well ? Admins should know whether or not to use this.

Reply:
Done. 
Also, apparently, `UWSGI_OPTIONS` is defined in the file (we already overwrite the uwsgi's schema description), so it's all much simpler (I've removed the extra function).